ingest/workflow/snakemake_rules/nextclade.smk

```diff
@@ -0,0 +1,73 @@
+rule nextclade_denvX:
+    """
+    For each type, classify into the appropriate subtype
+    """
+    input:
+        sequences="results/sequences_denv{x}.fasta",
+        dataset="../nextclade_data/denv{x}",
```

Comment:
Because this is reaching for files outside of the `ingest` directory, the instructions for running the workflow in the ingest README.md will no longer work.

You can follow the [pathogen-repo-guide to add a top level nextstrain-pathogen.yaml](https://github.com/nextstrain/pathogen-repo-guide/pull/29)

Reply:
Thanks for the link! Added the top level `nextstrain-pathogen.yaml` here: a7b8eeec74203c32d0f4dcb0319559c6e5dba743 and tried to update the docs accordingly: 37c800fd3beffe8f2e0cd47769b1310fdac1a301